LinearRegression/LinearRegressionBackend/DataProvider/DataConverter.cs

```diff
@@ -27,7 +28,33 @@ public static Vector<double> GetVector(double[,] imgPixelsMatrix)
 
         public static Matrix<double> ProcessInputImages(DirectoryInfo directory, IImageConverter converter)
         {
-            throw new NotImplementedException();
+            Vector<double> pixelVector;
+            Vector<double> labelVector;
+            Matrix<double> matrixOfPixels = null;
```

Comment:
Shape of the pixels matrix `X` is (n, m), where

- n: number of pixel which is calculated by `imageWidth * imageHeight`
- m: number of files in the folder